packages/sources/svelte/src/components/navigation/VtmnSearch/VtmnSearch.svelte

```diff
@@ -0,0 +1,109 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+
+  import { VtmnButton } from '@vtmn/svelte';
+  import { createEventDispatcher } from 'svelte';
+  import { VARIANTS, SIZES } from './enums';
+
+  /** @restProps { button } */
+
+  /**
+   * The variant of the button.
+   * @type {'default' | 'ghost' | 'persistent' | 'on-content'}
+   * @defaultValue 'default'
+   */
+  export let variant = VARIANTS.DEFAULT;
+
+  /**
+   * @type {boolean} disabled.
+   * @defaultValue false
+   */
+  export let disabled = false;
+
+  /**
+   * @type {'small' | 'medium'}
+   * @defaultValue 'medium'
+   */
+  export let size = SIZES.MEDIUM;
+
+  /**
+   * The value of the input
+   * @type {string}
+   */
+  export let value;
+
+  /**
+   * Custom classes to apply to the component.
+   * @type {string}
+   */
+  export let className;
+
+  const dispatch = createEventDispatcher();
+
+  const resetInputValue = () => {
+    value = '';
+  };
+
+  const onSearch = () => {
+    dispatch('search', {
+      text: value,
+    });
+  };
+
+  $: componentClass = cn(
+    'vtmn-search',
+    variant && `vtmn-search_variant--${variant}`,
+    size && `vtmn-search_size--${size}`,
+    className,
+  );
+</script>
+
+<form
+  class={componentClass}
+  action=""
+  role="search"
+  aria-controls="search-result"
+  on:submit|preventDefault={onSearch}
+>
+  <input
+    type="search"
+    {...$$restProps}
+    on:click
+    on:input
+    on:mouseover
+    on:mouseenter
+    on:mouseout
+    on:focus
+    on:blur
+    on:keydown
+    on:keypress
+    on:keyup
+    bind:value
+    {disabled}
+  />
+
+  <div class="vtmn-search_buttons">
+    <VtmnButton
+      iconAlone="close-line"
+      variant="ghost"
+      {disabled}
+      {size}
+      on:click={resetInputValue}
+      aria-label="close"
+      aria-disabled={disabled}
+    />
```

Comment:
```suggestion
  
```

No ghost variant

Reply:
If I delete ghost variant both buttons are blue background colors. In the vitamin css code ghost variant is present

Reply:
There's no "ghost" variant.
Not in our Storybook CSS, neither in Figma: https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vitamin---Web-UI-kit-(v0)?node-id=6527%3A15021.

But ghost are okay for button, don't worry. I think @GaspardMathon made a mistake.

Reply:
all right ! I keep ghost variant for buttons